tools/schemacode/bidsschematools/__main__.py

```diff
@@ -32,5 +33,50 @@ def export(ctx, schema, output):
             fobj.write(text)
 
 
+@cli.command()
+@click.argument("in_paths", nargs=-1, required=True)
+@click.option("--schema")
+@click.option("--dummy_paths", is_flag=True)
+@click.option("--bids_reference_root")
+@click.option("--schema_path")
+@click.option("--bids_version")
+@click.option("--report_path", is_flag=True)
+@click.option("--suppress_errors", is_flag=True)
+@click.option("accept_non_bids_dir", is_flag=True)
+@click.pass_context
+def validate(
+    ctx,
+    schema,
+    in_paths,
+    dummy_paths,
+    bids_reference_root,
+    schema_path,
+    bids_version,
+    report_path,
+    suppress_errors,
+    accept_non_bids_dir,
+):
```

Comment:
Okay, looking at most of these, I think we don't really want them. They seem to be aimed at future use cases that don't really exist right now. What I can see is:

```suggestion
@click.option("--schema")
@click.option("--report", is_flag=False, flag_value=True, default=False)
@click.pass_context
def validate(
    ctx,
    schema,
    in_paths,
    report,
):
```